java/compaction/compaction-job-execution/src/main/java/sleeper/compaction/job/execution/JavaCompactionRunner.java

```diff
@@ -71,7 +73,7 @@ public RowsProcessed compact(CompactionJob compactionJob, TableProperties tableP
         // Create a reader for each file
         List<CloseableIterator<Row>> inputIterators = createInputIterators(compactionJob, region, schema);
 
-        CloseableIterator<Row> mergingIterator = getMergingIterator(objectFactory, schema, compactionJob, inputIterators);
+        CloseableIterator<Row> mergingIterator = getMergingIterator(objectFactory, schema, compactionJob, inputIterators, tableProperties.get(FILTERS_CONFIG));
```

Comment:
It doesn't look like this is tested. That can go in JavaCompactionRunnerIteratorIT.